docs/reference/experiments.md

```diff
@@ -1,11 +1,25 @@
 # Experiments
 
-The Slack CLI has an experiment flag, behind which we put features under development and may cause breaking changes. These features are fleeting and many will not work without a development instance, but are available for use "at your own risk."
+The Slack CLI has an experiment (`-e`) flag behind which we put features currently under development. These features may be fleeting, may not be perfectly polished, and many will not work without a development instance - but we have made them available for use "at your own risk."
 
 ## Available experiments
 
-The following is a list of currently available experiments. We may remove an experiment once the feature is released.
+The following is a list of currently available experiments. We'll remove experiments from this page if we decide they are no longer needed or once they are released, in which case we'll make an announcement about the feature's general availability in the [developer changelog](https://docs.slack.dev/changelog).
 
 * `bolt-install`: enables creating, installing, and running Bolt projects that manage their app manifest on app settings (remote manifest).
     * `slack create` and `slack init` now set manifest source to "app settings" (remote) for Bolt JS & Bolt Python projects ([PR#96](https://github.com/slackapi/slack-cli/pull/96)).
 * `read-only-collaborators`: enables creating and modifying collaborator permissions via the `slack collaborator` commands.
+* `experiment bolt-install`: enables creating, installing, and running Bolt projects that manage their app manifest on app settings (remote manifest).
```

Comment:
suggestion: We can remove this line because it's documented at the top as `bolt-install`:
```suggestion
```